File: parcial02/punto02/java/batch/src/main/java/co/edu/eafit/st0257/s20181/Card.java
package co.edu.eafit.st0257.s20181;

public class Card {
   private static int nro = 0;
   private int nCard;
   public Card() {
      nCard = Card.nro++;
   }
   public String toString() {
      return "Card: " + nCard;
   }
}

File: parcial02/punto02/c++/src/card.cpp
#include "card.h"

int Card::nro = 0;

Card::Card() : nCard(Card::nro++) { }

std::ostream &operator<<(std::ostream& strm, const Card card) {
  strm << "Card: " << card.nCard << std::endl;
  return strm;
}
